lms/static/js/spec/main.js

```diff
@@ -325,6 +326,10 @@
                 exports: 'js/ccx/schedule',
                 deps: ['jquery', 'underscore', 'backbone', 'gettext', 'moment']
             },
+            'js/ccx/async_grade_book_download': {
+                exports: 'js/ccx/async_grade_book_download',
+                deps: ['jquery', 'underscore', 'backbone', 'gettext']
+            },
```

Comment:
FYI, this section isn't necessary, since `async_grade_book_download.js` already uses the `define` function. The `shim` section of this config is for backwards compatibility with JS files that do *not* declare their dependencies using the `define` function, and you already do that.